website/content/en/preview/AWS/provisioning.md

```diff
@@ -6,6 +6,40 @@ description: >
   Learn AWS-specific Karpenter provisioning settings
 ---
 
+Provisioner settings specific to Karpenter for the AWS cloud provider are described here.
+The following example shows optional and required settings for a Karpenter provisioner for AWS:
+
+
+```yaml
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata:
+  name: default
+spec:
+  requirements:                      
+    - key: karpenter.sh/capacity-type         # optional, on-demand by default; spot if both are listed
+      operator: In
+      values: ["spot"]
+  limits:   
+    resources:
+      cpu: 1000                               # optional, recommended to limit total cluster CPU
+      memory: 1000Gi                          # optional, recommended to limit total cluster memory
+  provider:
+    subnetSelector:                           # required
+      karpenter.sh/discovery: ${CLUSTER_NAME}
+    securityGroupSelector:                    # required
+      karpenter.sh/discovery: ${CLUSTER_NAME}
+    AMIFamily: AmazonLinux2                   # optional
+    InstanceProfile: MyInstanceProfile        # optional, set in [Getting Started]({{<ref "../getting-started/getting-started-with-eksctl/" >}}), but can overrride here
+    Tags:                                     # optional, override default tags or set your own
+      MySpecialTag: true

```

Comment:
It's probably better to use an actual `string` here rather than a `bool` where the type is ambiguous. I'm not sure what a good value is here, so I just put `foo`.  

```suggestion
      MySpecialTag: foo
```

Reply:
Done